liquibase-core/src/main/java/liquibase/Liquibase.java

```diff
@@ -2337,52 +2337,10 @@ public void setChangeLogParameter(String key, Object value) {
-        setChangeLogParameter("database.autoIncrementClause", database.getAutoIncrementClause(null, null, null, null));
-        setChangeLogParameter("database.currentDateTimeFunction", database.getCurrentDateTimeFunction());
-        setChangeLogParameter("database.databaseChangeLogLockTableName", database.getDatabaseChangeLogLockTableName());
-        setChangeLogParameter("database.databaseChangeLogTableName", database.getDatabaseChangeLogTableName());
-        setChangeLogParameter("database.databaseMajorVersion", database.getDatabaseMajorVersion());
-        setChangeLogParameter("database.databaseMinorVersion", database.getDatabaseMinorVersion());
-        setChangeLogParameter("database.databaseProductName", database.getDatabaseProductName());
-        setChangeLogParameter("database.databaseProductVersion", database.getDatabaseProductVersion());
-        setChangeLogParameter("database.defaultCatalogName", database.getDefaultCatalogName());
-        setChangeLogParameter("database.defaultSchemaName", database.getDefaultSchemaName());
-        setChangeLogParameter("database.defaultSchemaNamePrefix", StringUtil.trimToNull(database.getDefaultSchemaName()) == null ? "" : "." + database.getDefaultSchemaName());
-        setChangeLogParameter("database.lineComment", database.getLineComment());
-        setChangeLogParameter("database.liquibaseSchemaName", database.getLiquibaseSchemaName());
-        setChangeLogParameter("database.liquibaseTablespaceName", database.getLiquibaseTablespaceName());
-        setChangeLogParameter("database.typeName", database.getShortName());
-        setChangeLogParameter("database.isSafeToRunUpdate", database.isSafeToRunUpdate());
-        setChangeLogParameter("database.requiresPassword", database.requiresPassword());
-        setChangeLogParameter("database.requiresUsername", database.requiresUsername());
-        setChangeLogParameter("database.supportsForeignKeyDisable", database.supportsForeignKeyDisable());
-        setChangeLogParameter("database.supportsInitiallyDeferrableColumns", database.supportsInitiallyDeferrableColumns());
-        setChangeLogParameter("database.supportsRestrictForeignKeys", database.supportsRestrictForeignKeys());
-        setChangeLogParameter("database.supportsSchemas", database.supportsSchemas());
-        setChangeLogParameter("database.supportsSequences", database.supportsSequences());
-        setChangeLogParameter("database.supportsTablespaces", database.supportsTablespaces());
-    }
-
-    private LockService getLockService() {
-        return LockServiceFactory.getInstance().getLockService(database);
-    }
-
     public void setChangeExecListener(ChangeExecListener listener) {
         this.changeExecListener = listener;
     }
 
-    public void setChangeLogSyncListener(ChangeLogSyncListener changeLogSyncListener) {
-        this.changeLogSyncListener = changeLogSyncListener;
-    }
```

Comment:
Should we keep this and mark it as deprecated in case someone is using it? I am not suggesting that this is necessary, just asking for opinions.

Reply:
I am restoring it. Should have only removed unused private methods, this one was too much .